### PR TITLE
#80 updated readme

### PR DESCRIPTION
**Pull request by tvirat**
https://github.com/Jai0212/Cash-App-Bias-Busters/issues/80
https://github.com/Jai0212/Cash-App-Bias-Busters/issues/77

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -66,11 +65,26 @@ npm run dev
...
+
+  - **Packages Used**:
+    - `numpy`: For numerical computations and matrix operations used in model analysis.
+    - `pandas`: For handling and preprocessing data.
+    - `scikit-learn`: For building and training the machine learning models.
+    - `scipy`: For advanced statistical computations required in analysis.
+
+- **Frontend**: Built using React and Vite, the frontend is responsible for the interactive dashboard and visualizations. Hosted on **Netlify**.
+
+  - **Packages Used**:
```
Comment (Jai0212) [inline README.md:80]: sweet alert and axiosRetry missing

File: README.md [hunk 2]
```diff
@@ -66,11 +65,26 @@ npm run dev
...
-- <b>Backend:</b> Built using <b>Python</b> with the <b>Flask</b> framework, the backend handles data processing and bias detection. Hosted on <b>Render</b>.
-- <b>Frontend:</b> Built using <b>React</b> and Vite, the frontend is responsible for the interactive dashboard and visualizations. Hosted on <b>Netlify</b>.
-- <b>Database:</b> We use <b>MySQL</b> to store all the data uploaded by users. The database is hosted on Aiven.
-- <b>Fairlearn:</b> This Python library is used for detecting and assessing fairness in ML models.
-- <b>Chart.js:</b> Used to render dynamic and interactive graphs and charts to visualize bias metrics.
+### Technical Implementation Details
+
+- **Backend**: Built using Python with the Flask framework, the backend handles data processing and bias detection. Hosted on **Render**.
+
+  - **Packages Used**:
```
Comment (Jai0212) [inline README.md:72]: mysql-connector and pytest missing

**Review by Jai0212 — CHANGES_REQUESTED** [review]
Mention CLEAN ARCHITECURE (by layer) and TESTING as well

image
The part above can be written better explaining what our thing actually does

In the how to run, use, start from the cloning of the project

The labels at the top are good but some can be made different colour so its eye catching.

Technical details should come before how to use

More comments are given in the code

**Comment by Jai0212** [comment]
Another thing I forgot to mention in the previous CSS PR, the time selected should be blue like before for better clarity.

The space between the word "demographics" and the stuff below should be increased. I think that whole sections needs to be spaced out more. very crammed at the moment.

image

**Comment by tvirat** [comment]
> Mention CLEAN ARCHITECURE (by layer) and TESTING as well
> 
> image. The part above can be written better explaining what our thing actually does
> 
> In the how to run, use, start from the cloning of the project
> 
> The labels at the top are good but some can be made different colour so its eye catching.
> 
> Technical details should come before how to use
> 
> https://cash-app-bias-busters.onrender.com/ - put our backend link also near the frotnend link and mention the backend can take up to a minute to startup
> 
> More comments are given in the code

I have added the mentioned packages and have addressed the other mentioned issues.

**Review by Jai0212 — COMMENTED** [review]
Looks good

**Pull request merged**